### ISSUE TITLE
Store on WordPress.com: Deferred Stripe Connect Accounts No Longer Available after 2020 May

### ISSUE THREAD

**Issue by allendav** [Automattic/wp-calypso#31082]
p7bbVw-38w-p2

Starting in July, Stripe Connect accounts created using the deferred flow will be unable to accept payments from buyers until the seller/merchant completes Stripe on-boarding.  This is due to the new [CDD (Customer Due Diligence) Rule](https://www.fincen.gov/resources/statutes-and-regulations/cdd-final-rule).

The existing flow prompts users to verify their email but allows them to accept payments. This will not be the case after July and we can expect some merchants to be surprised that the Stripe payment method isn't working. To avoid a potential deluge of support inquiries, we should update the prompts in Store on WordPress.com Settings for the Stripe Connect flow.

What we do today

- On Store on WPCOM, after creating a deferred account, we display “Stripe is now connected. An email from Stripe has been sent to EMAIL to activate the account.”
- Or, if the merchant just connected an already activated account we instead display “Stripe is now connected! You can start accepting payments!”
- Or, if the merchant is simply browsing their Stripe settings, and if they haven’t finished activating a deferred account (based on the payouts_enabled flag in their account), we display “Check email to activate account.”.
- Lastly, if the merchant is simply browsing their Stripe settings and they have finished activation, we present simply “Activated.”

What we could to do to handle CDD changes properly

- On Store on WPCOM, after creating a deferred account, we should display “Stripe is now connected but is unable to receive payments. An email from Stripe has been sent to EMAIL with some additional steps you’ll need to complete before you can receive payments.”
- Or, if the merchant just connected an already activated and verified account we instead display “Stripe is now connected and ready to receive payments!”
- If the merchant just connected an already activated account that requires additional identity information or verification, we should display “Stripe is now connected but is unable to receive payments. An email from Stripe has been sent to EMAIL with some additional steps you’ll need to complete before you can receive payments.”
- Or, if the merchant is simply browsing their Stripe settings, and if they haven’t finished activating a deferred account or that account requires additional identity information or verification, we display “Stripe is connected but unable to receive payments. An email from Stripe has been sent to EMAIL with some additional steps you’ll need to complete before you can receive payments.”.
- Lastly, if the merchant is simply browsing their Stripe settings and they have finished activation and the account requires no additional identity information or verification, we present simply “Connected and able to receive payments!”

cc @timmyc @LevinMedia @dechov @pmaiorana @jameskoster

**Comment by allendav** [comment]
e.g. here https://github.com/Automattic/wp-calypso/blob/59bdfeeb97eda4266ad39410cb0a074d2c88dbc8/client/extensions/woocommerce/app/settings/payments/stripe/payment-method-stripe-connected-dialog.js#L100

**Comment by allendav** [comment]
Note: We will probably need to also make changes in redux 

https://github.com/Automattic/wp-calypso/blob/59bdfeeb97eda4266ad39410cb0a074d2c88dbc8/client/extensions/woocommerce/state/sites/settings/stripe-connect-account/actions.js#L172

and in WooCommerce Services to return more than just `payouts_enabled` for the sake of determining if the account is fully activated - we will also need `charges_enabled`

https://github.com/Automattic/woocommerce-services/blob/master/classes/class-wc-connect-stripe.php#L88

See also https://stripe.com/docs/connect/identity-verification-api#determining-if-identity-verification-is-needed

**Comment by dechov** [comment]
We may be able to link the merchant straight to the activation form rather than having them look through their email – this was a suggestion from a previous version (now revised) of [this doc](https://stripe.com/docs/connect/updated-requirements-standard): "you can share activation links with users to help facilitate account activations". I believe this link would be to https://dashboard.stripe.com/account/details

**Comment by allendav** [comment]
Note: Hold for now. See referenced P2 post.

**Comment by allendav** [comment]
Stripe will be ending support on May 31 for this flow. Re-opening.

**Comment by nerdysandy** [comment]
We're sunsetting Store by Feb,2021. Closing it in favour of that
